.gitmodules

```diff
@@ -14,6 +14,7 @@
 [submodule "deps/VectorSimilarity"]
 	path = deps/VectorSimilarity
 	url = https://github.com/RedisAI/VectorSimilarity.git
+	branch = main
```

Comment:
should we perhaps have some marker of the VecSim related tests and have a separate set of tests dedicated to this?

Isn't there a big risk of blocking too many PR merges if there is a bug in main that one cannot fix here?

Reply:
redisearch is not going to point to vecsim main
This is a temporary change to allow me to utilize the CI to run against different platforms